packages/data-stores/src/woocommerce-eligibility/constants.ts

```diff
@@ -0,0 +1,41 @@
+export const STORE_KEY = 'automattic/woocommerce-eligibility';
+
+export const eligibilityHolds = {
```

Comment:
Do we really need to export this one? I'd always encourage not exporting something when it's not necessary to be consumed from outside.